tests/model_registry/rbac/test_mr_rbac_sa.py

```diff
@@ -0,0 +1,145 @@
+# AI Disclaimer: Google Gemini 2.5 pro has been used to generate a majority of this code, with human review and editing.
+import pytest
+from typing import Self, Dict, Any
+from simple_logger.logger import get_logger
+
+# Framework / Client specific imports
+from model_registry import ModelRegistry as ModelRegistryClient
+from mr_openapi.exceptions import ForbiddenException
+
+from utilities.constants import DscComponents, Protocols  # Need Protocols.HTTPS
+from tests.model_registry.constants import MR_NAMESPACE
+
+
+LOGGER = get_logger(name=__name__)
+
+
+# --- Helper to build client args ---
+# Based on the 'model_registry_client' fixture
+def build_mr_client_args(rest_endpoint: str, token: str, author: str) -> Dict[str, Any]:
+    """Builds arguments for ModelRegistryClient based on REST endpoint and token."""
+    try:
+        server, port = rest_endpoint.split(":")
+
+        # Conftest example uses is_secure=False - this implies TLS verification might be off
+        # Adjust if your environment requires strict TLS verification
+        return {
+            "server_address": f"{Protocols.HTTPS}://{server}",
+            "port": port,
+            "user_token": token,
+            "is_secure": False,  # Match conftest example (disable TLS verification)
+            "author": author,
+        }
+    except Exception as e:
```

Comment:
This seems not necessary. If you don't get rest_endpoint, may be in that fixture raise exception?